com.liferay.blade.cli/src/com/liferay/blade/cli/gradle/GradleTooling.java

```diff
@@ -71,7 +71,8 @@ public static File findLatestAvailableArtifact(
 
 			connection = connector.connect();
 
-			connection.newBuild().forTasks("copyDep").run();
+			connection.newBuild().forTasks("copyDep").withArguments(
```

Comment:
Lets make this "refresh" only be used when the users specifies a "refresh" option
`blade create -r -t portlet` where `-r` means "refresh"